documentation/en/source/pages/machine-images/builders/builders-openshift.rst

```diff
@@ -38,6 +38,14 @@ If you are using JSON:
 	  ]
 	}
 
+Building a Machine Image
+------------------------
+
+For building an image, the valid keys are:
+
+* ``type`` (mandatory): a string providing the machine image type to build. Default builder type for OpenShift: ``OpenShift``. To get the available builder type, please refer to :ref:`command-line-format`.
+* ``entrypoint`` (mandatory): a string describing the command to launch at OpenShift container start in ``exec`` form.
+
```

Comment:
Line 80, the builder is not up to date with this information, could you update it please ? (for some reason I can only comment on your added lines)